workers/unity/Assets/Plugins.meta

```diff
@@ -1,5 +1,5 @@
 fileFormatVersion: 2
-guid: 34c0720497daf334290b4cb0ac5f7db2
+guid: a2bb714765233254eb4d9463c199df91
```

Comment:
? Why is this file changed?

Reply:
Discussed offline - this is created due to Rider plugins every time we open the project. However, since nothing else is in the folder and hence not checked in, we have a `.meta` file and no folder.

Solution - add a dummy file to the Plugins directory so it can be committed and the meta file won't change 😄